src/xb-node.c

```diff
@@ -23,6 +23,30 @@ typedef struct {
 G_DEFINE_TYPE_WITH_PRIVATE (XbNode, xb_node, G_TYPE_OBJECT)
 #define GET_PRIVATE(o) (xb_node_get_instance_private (o))
 
+/**
+ * XbNodeIter:
+ *
+ * A XbNodeIter structure represents an iterator that can be used
+ * to iterate over the attributes of a #XbNode. XbNodeIter
+ * structures are typically allocated on the stack and then initialized
+ * with %xb_node_attr_iter_init().
+ *
+ * The iteration order of a #XbNodeIter is not defined.
+ */
+
+typedef struct
+{
+  XbNode 	*node;
+  guint8	position;
```

Comment:
Alignment is off for `position` relative to `node`. Either don’t align any of the members’ names, or align all of them.

Reply:
Weird, it looks aligned in my editor... I added an extra space, maybe that helps (it at least aligns the other entries with the name after the asterisk)

Reply:
I think the other lines use tabs, not spaces.